v20.1/explain-analyze.md

```diff
@@ -87,6 +117,27 @@ To view the [DistSQL Plan Viewer](#distsql-plan-viewer), point your browser to t
 
 <img src="{{ 'images/v20.1/explain-analyze-distsql-plan.png' | relative_url }}" alt="EXPLAIN ANALYZE (DISTSQL)" style="border:1px solid #eee;max-width:100%" />
 
+
+### `EXPLAIN ANALYZE (DEBUG)`
+
+Use the [`DEBUG`](#debug-option) option to generate a ZIP file containing files with information about the query and the database objects referenced in the query. For example:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> EXPLAIN ANALYZE SELECT l_shipmode, AVG(l_extendedprice) FROM lineitem GROUP BY l_shipmode;
```

Comment:
Sorry, I just now noticed this in the doc - this should be `EXPLAIN ANALYZE (DEBUG)`

Reply:
Good catch!

Reply:
See #7187.